MeshLib/NodePartitionedMesh.h

```diff
@@ -96,11 +101,11 @@ class NodePartitionedMesh : public Mesh
         bool isGhostNode(const std::size_t node_id) const
         {
             if(node_id < _n_active_base_nodes)
-                return true;
+                return false;
```

Comment:
This line and the following if/else's do change the logic completly. Is it a bug-fix or something?